packages/wagmi/src/utils/defaultWagmiCoreConfig.ts

```diff
@@ -76,7 +76,7 @@ export function defaultWagmiConfig({
     ...auth
   }
 
-  if (mergedAuth.email || mergedAuth.socials) {
+  if (mergedAuth.email || mergedAuth.socials?.length) {
```

Comment:
Why don't we allow users to have email login if they don't set social options? Cannot they use only email?